misc/nsis/qutebrowser.nsi

```diff
@@ -0,0 +1,179 @@
+!define MSI32_041 "{66F9576D-0DB5-475D-9C25-E0511580C897}"
+!define MSI32_050 "{EDB54F4D-7A00-47AE-9808-E59FF5E79136}"
+!define MSI32_051 "{EEF03487-BC9F-4EA4-A5A1-E9CF5F9E1FB6}"
+!define MSI32_060 "{F9FECA24-95DA-4E46-83E3-A805E5B1CE06}"
+!define MSI32_061 "{F1A0F4B9-CCA3-4B07-8ADB-16BC81530440}"
+!define MSI32_062 "{43F5E4C5-FF96-4676-B027-5AD63D3871AB}"
+!define MSI32_070 "{558FF39C-CA5F-4E2A-87D2-90963FCBC424}"
+!define MSI32_080 "{9DF540E2-4F8C-46A4-A27F-43BD0558CC42}"
+!define MSI32_081 "{EA0FB6B1-83AF-4F16-8B89-645B587D90FD}"
+!define MSI32_082 "{F849A0B2-301C-435D-9CC0-9651938FCA6F}"
+!define MSI32_084 "{9331D947-AC86-4542-A755-A833429C6E69}"
+!define MSI32_090 "{AD967987-7777-4095-A03A-3F2EE8968D9E}"
+!define MSI32_091 "{87F05B8A-2238-4D86-82BB-EC8B4CE97E78}"
+!define MSI32_100 "{07B85A0B-D025-4B4B-B46D-BC9B02912835}"
+!define MSI32_101 "{9F05D9E4-D049-445E-A489-A7DC0256C774}"
+!define MSI64_010 "{A8191862-28A7-4BB0-9532-49AD5CFFFE66}"
+!define MSI64_011 "{1C476CC1-A171-48B7-A883-0F00F4D301D3}"
+!define MSI64_012 "{ADA727AC-9DDD-4F03-93B7-BAFE950757BE}"
+!define MSI64_013 "{64949BFF-287A-4C16-A5F3-84A38A6703F1}"
+!define MSI64_014 "{63F22761-D886-4FDD-93F4-7543265E9FF7}"
+!define MSI64_020 "{80BE09C6-347F-4121-98D3-1E4363C3CE6B}"
+!define MSI64_021 "{2D86F472-DD52-40A1-8FE0-90550D674554}"
+!define MSI64_030 "{53DED10D-C609-406F-959E-C1B52A518561}"
+!define MSI64_040 "{B9535FDF-7A9E-4AED-BA1E-BEE5FFCBC311}"
+!define MSI64_041 "{DAE1309A-FE7D-46E5-B488-B437CC509DF9}"
+!define MSI64_050 "{DC9ECE64-F8E5-4BCB-BCFF-BE4ADCEF2655}"
+!define MSI64_051 "{26AED286-23BD-49FF-BD9C-7C0DC4467BD7}"
+!define MSI64_060 "{3035744D-2390-4D5E-ACAD-905E72B9EBEC}"
+!define MSI64_061 "{0223F48F-93A8-4985-BCFF-328E5A9D97D5}"
+!define MSI64_062 "{95835A82-A9C2-4924-87DF-E03D910E3400}"
+!define MSI64_070 "{61D1AC75-7ECD-45FF-B42B-454C056DB178}"
+!define MSI64_080 "{92D1C65C-1338-4B11-B515-6BD5B1FF92D9}"
+!define MSI64_081 "{AF7AC009-FB82-48F6-9439-6E46AEB60DBF}"
+!define MSI64_082 "{CC316D68-5742-4C2B-98EC-4ADF06A19B84}"
+!define MSI64_084 "{633F41F9-FE9B-42D1-9CC4-718CBD01EE11}"
+!define MSI64_090 "{5E3E7404-D6D7-4FF1-846A-F9BBFE2F841A}"
+!define MSI64_091 "{3190D3F6-7B24-47DC-88E7-99280905FACF}"
+!define MSI64_100 "{7AA6530C-3812-4DC5-9A30-E762BBDDF55E}"
+!define MSI64_101 "{B0104B85-8229-49FB-8606-275A90ACC024}"
+
+; Set PLATFORM - default x64
+!ifdef X86
+  !define PLATFORM "Win32"
+  !define ARCH "x86"
+  !define SUFFIX "win32"
+!else
+  !define PLATFORM "Win64"
+  !define ARCH "x64"
+  !define SUFFIX "amd64"
+!endif
+
+; If not defined, get VERSION from PROGEXE. Set DIST_DIR accordingly.
+!ifndef VERSION
+  !define /ifndef DIST_DIR ".\..\..\dist\${PRODUCT_NAME}-${ARCH}"
+  !getdllversion "${DIST_DIR}\${PROGEXE}" expv_
+  !define VERSION "${expv_1}.${expv_2}.${expv_3}"
+!else
+  !define /ifndef DIST_DIR ".\..\..\dist\${PRODUCT_NAME}-${VERSION}-${ARCH}"
+!endif
+
+; Pack the exe header with upx, unless NOUPX is defined.

```

Comment:
Probably not. The size gain is minimal but so is the overhead (a/v scanners have no problem with files created by unmodified upx). Should we remove it completely or keep an optional `UPX` define?

Reply:
FWIW I don't have UPX installed currently, and got a failing build because of that. I think I'd rather not use it at all, if it doesn't make a big difference.

Reply:
Thats strange. If UPX is missing it just displays a warning and continues to use the unpacked header. Anyway, i disable the default use of UPX in b1a33d6.